scripts/release.py

```diff
@@ -359,6 +361,10 @@ def prs_by_email(start_ref, end_ref):
     The dictionary is alphabetically ordered by email address
     The pull request list is ordered by merge date
     """
+    # `emails` maps from other_emails to primary email, based on people.yaml.
+    people = yaml.safe_load(requests.get(PEOPLE_YAML).text)
+    emails = { em: person['email'] for person in people.itervalues() for em in person.get('other_emails', []) }
```

Comment:
would we have to change people.yaml to have this 'other_emails' field? Right now you're the only person who uses it:

```
>>> import yaml
>>> PEOPLE_YAML = "https://raw.githubusercontent.com/edx/repo-tools/master/people.yaml"
>>> people = yaml.safe_load(requests.get(PEOPLE_YAML).text)
>>> emails = { em: person['email'] for person in people.itervalues() for em in person.get('other_emails', []) }
>>> emails
{'ned@nedbatchelder.com': 'ned@edx.org', 'ned@python.org': 'ned@edx.org'}
```

I think we're going to have to find people's usernames rather than emails--probably by using github's api, then using `people.yaml` to find their emails